src/error.rs

```diff
@@ -214,14 +214,12 @@ impl From<herror::Error> for CmdError {
 
 impl From<wderror::WebDriverError> for CmdError {
     fn from(e: wderror::WebDriverError) -> Self {
-        if let wderror::WebDriverError {
-            error: wderror::ErrorStatus::NoSuchElement,
-            ..
-        } = e
-        {
-            CmdError::NoSuchElement(e)
-        } else {
-            CmdError::Standard(e)
+        match e {
```

Comment:
Maybe undo this change entirely (make it be `if let` again) now that we're no longer looking for script timeout?